packages/adapter/src/chains/oak.ts

```diff
@@ -25,8 +25,8 @@ export interface AutomationPriceTriggerParams {
 }
 
 export enum OakAdapterTransactType {
-  PayThroughRemoteDerivativeAccount,
-  PayThroughSoverignAccount,
+  PayThroughRemoteDerivativeAccount = "PayThroughRemoteDerivativeAccount",
+  PayThroughSoverignAccount = "PayThroughSoverignAccount",
 }
```

Comment:
These values are called "instructionSequence" in blockchain. We should update the type to "InstructionSequenceType" for consistency.